### PR TITLE
SSL Extension - fixes validator existence bug

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by Cedev** [comment]
Where are the changes for the ssl and scaling group extensions?

File: a10_openstack_lib/resources/a10_scaling_group.py [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Copyright (C) 2016 A10 Networks Inc. All rights reserved.
```
Comment (Cedev) [inline a10_openstack_lib/resources/a10_scaling_group.py:1]: a10-openstack-lib needs the tox rule to check for Apache 2.0 license headers.

**Comment by ghost** [comment]
RE: "changes for the SSL and scaling group extensions" - these extensions were merged many moons ago. This commit fixes some brokenness that was introduced w/ Openstack.

**Comment by Cedev** [comment]
The type names need to be changed in a10-neutronclient too.

**Comment by ghost** [comment]
See https://github.com/a10networks/a10-neutronclient/pull/9 for fixed type names.

**Comment by ghost** [comment]
check

**Comment by ghost** [comment]
check

**Comment by Cedev** [comment]
Tox isn't catching the missing apache 2.0 headers

File: a10_openstack_lib/resources/a10_scaling_group.py [hunk 2]
```diff
@@ -257,7 +258,7 @@
         'reactions': {
             'allow_post': True,
             'allow_put': True,
-            'convert_list_to': lambda attr: attr.convert_kvp_list_to_dict,
+            'convert_to': lambda attr: attr.convert_to_list,
```
Comment (Cedev) [inline a10_openstack_lib/resources/a10_scaling_group.py]: Does this work?

**Review by Cedev — CHANGES_REQUESTED** [review]

**Review by Cedev — APPROVED** [review]

File: a10_openstack_lib/resources/a10_certificate.py [hunk 3]
```diff
@@ -180,7 +180,7 @@ def f(data, options):
 
 
 VALIDATORS = {
-    'type:a10_float': lambda validators: validate_float,
-    'type:a10_reference': lambda validators: validate_reference,
-    'type:a10_nullable': validate_nullable
+    'a10_type:float': lambda validators: validate_float,
+    'a10_type:reference': lambda validators: validate_reference,
+    'a10_type:nullable': validate_nullable
```
Comment (Cedev) [inline a10_openstack_lib/resources/a10_certificate.py:185]: How did you slop this past me?!?!

**Pull request merged**